extensions/adobe/experience/adcloud/search/account-all.example.1.json

```diff
@@ -0,0 +1,15 @@
+{
+    "@id": "100001061",
```

Comment:
`@id` is special in JSON-LD.  You probably want to be using `xdm:id`

Reply:
Fixed to use xdm:id